test/unit/matchers/matchers_test.rb

```diff
@@ -0,0 +1,118 @@
+require "minitest/autorun"
+require "rspec/matchers"
+require "rspec/expectations"
+require "matchers/matchers"
+
+require "pry"
+
+describe 'inspec matchers' do
+  describe 'cmp matcher' do
+    include RSpec::Matchers
+
+    def rspec_expect value, &block
```

Comment:
This probably needs in-code comments to explain what is going on. We shouldn't assume maintainers have any particular knowledge of RSpec.

Reply:
fixed